src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -801,6 +803,7 @@ function TTYGViewCtrl(
 
     const buildRepositoryList = () => {
         $scope.activeRepositoryList = $repositories.getReadableRepositories()
+            .filter((repo) => repo.type === 'graphdb')
```

Comment:
You can create another method in the repositories service like getReadableGraphdbRepositories or something and reuse it

Reply:
Done